plugins/woocommerce/client/blocks/assets/js/blocks/checkout/styles/style.scss

```diff
@@ -33,7 +33,7 @@
 	}
 }
 .wp-block-woocommerce-checkout {
-	margin: 0;
+	margin: 0 auto;
```

Comment:
Let's add this change only for the impacted themes for now.
```
.theme-twentytwenty .wp-block-woocommerce-checkout, 
.theme-twentytwentyone .wp-block-woocommerce-checkout,   {
margin: 0 auto;
}
```
You'll find a similar customization at the bottom of the file.